gradle-scripts/test.gradle

```diff
@@ -2,5 +2,8 @@ test {
     useJUnitPlatform {
         includeEngines 'junit-jupiter', 'junit-vintage'
```

Comment:
Why do you need `junit-vintage`? Why not use only new junit?

Reply:
it's not needed: 
ab71ee7